test/index.html

```diff
@@ -13,6 +13,7 @@
 
   <body>
     <div id="root"></div>
+    <div id="input" style="position: fixed; width: 100%; top: 0; right: 0;"></div>
```

Comment:
And remove this because we want the input/button along with the fixed div container (not the input) to be added by the `init` function.